types/adobe__pdfservices-node-sdk/operation/rotate-pages-operation.d.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * An operation that allows rotation of specific pages in a PDF file.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        RotatePages = PDFServicesSdk.RotatePages,
+ *        rotatePagesOperation = RotatePages.Operation.createNew();
+ *
+ *  rotatePagesOperation.setInput(PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/rotatePagesOperationInput.pdf',
+ *  	RotatePages.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/rotate-pages-operation.d.ts;
  Line: 15;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.